internal/core/media.go

```diff
@@ -144,12 +124,12 @@ func (c *Community) GetVideoSubtitle(ctx context.Context, mediaId string) (strin
 	return fileKey, status, nil
 }
 
-func (c *Community) SaveMedia(ctx context.Context, userId string, data []byte, fileExt string) (int64, string, error) {
+func (c *Community) SaveMedia(ctx context.Context, userId string, data []byte, fileExt string) (int64, error) {
 	opts := &blob.WriterOptions{}
 	// upload cloud oss
 	fileKey := uuid.New().String()
-
-	if isTranscoding(fileExt) {
+	c.xLog.Info(fileKey)
+	if strings.Contains(fileExt, "video") {
```

Comment:
@xhyqaq  All video types have this prefix?

Reply:
yes，all video is  video/xxx